messaging/src/main/java/org/axonframework/messaging/Message.java

```diff
@@ -97,28 +97,28 @@ public interface Message<P> {
     Class<P> getPayloadType();
 
     /**
-     * Returns a copy of this {@code Message message (implementation)} with the given {@code metaData}.
+     * Returns a copy of {@code this Message (implementation)} with the given {@code metaData}.
```

Comment:
I think the `{@code` could be left out right? :thinking: It's not code

Reply:
You're right, only the `Message` could be `code`. I'll change it.